### PR TITLE
allows leaderboard to be browsed by product

### PR DESCRIPTION
**Pull request by owocki**
##### Description
allows leaderboard to be browsed by product

##### Refers/Fixes

https://gitcoincore.slack.com/archives/CAXQ7PT60/p1573916653291000?thread_ts=1572662544.214300&cid=CAXQ7PT60

##### Testing

tested locally

### CODE REVIEW

File: app/marketing/management/commands/assemble_leaderboards.py [hunk 1]
```diff
@@ -372,70 +374,78 @@ class Command(BaseCommand):
...
+                for kt in KudosTransfer.objects.send_success().filter(network='mainnet'):
+                    sum_kudos(kt)
+
+            # set old LR as inactive
+            with transaction.atomic():
+                lrs = LeaderboardRank.objects.active()
+                lrs.update(active=False)
+
+                # save new LR in DB
+                for key, rankings in ranks.items():
```
Comment (owocki) [inline app/marketing/management/commands/assemble_leaderboards.py]: all indentation changes from here on out

File: app/marketing/management/commands/assemble_leaderboards.py [hunk 2]
```diff
@@ -290,6 +290,8 @@ def sum_tip_helper(t, time, index_term, val_usd):
 
 def sum_kudos(kt):
     val_usd = kt.value_in_usdt_now
+    if not kt.kudos_token_cloned_from:
+        return
```
Comment (danlipert) [inline app/marketing/management/commands/assemble_leaderboards.py]: hmm.. what does this logic do here?

Comment (owocki) [reply app/marketing/management/commands/assemble_leaderboards.py]: there is a kudos transfer that has a null `kudos_token_cloned_from` and this logic prevents an exception here.

Comment (danlipert) [reply app/marketing/management/commands/assemble_leaderboards.py]: ah nice thanks for the info

**Comment by thelostone-mc** [comment]
@owocki looks like we've got conflicts cause of the indentation 😓  
would you be able resolve those?

**Comment by danlipert** [comment]
@owocki can you take another look at this PR - seems like the tests for the leaderboard are broken. I tried a quick fix but failed

**Comment by owocki** [comment]
yay! fixed. merging

**Pull request merged**